sdk/logical/events.go

```diff
@@ -67,3 +67,39 @@ func SendEvent(ctx context.Context, sender EventSender, eventType string, metada
 	}
 	return sender.SendEvent(ctx, EventType(eventType), ev)
 }
+
+// EventReceivedBexpr is used for evaluating boolean expressions with go-bexpr.
+type EventReceivedBexpr struct {
+	EventType         string `bexpr:"event_type"`
+	Operation         string `bexpr:"operation"`
+	SourcePluginMount string `bexpr:"source_plugin_mount"`
+	FullSecretPath    string `bexpr:"full_secret_path"`
```

Comment:
Any reason not to use `data_path` for consistency? I know `data_path` starts its life from the plugin as not a full path, but from an end user's point of view it always ends up being a full path once we've updated it right?

Reply:
Good call. I was mostly just copy-pasting from the RFC, but we should strive for consistency.